src/components/popup.tsx

```diff
@@ -74,37 +89,44 @@ export class Popup extends React.Component<PopupProps, PopupState> {
 
     render(): JSX.Element {
         const { headerText, bodyText, primaryBtnText, secondaryBtnText,
-            primaryBtnStyle, secondaryBtnStyle, validationErrors, children } = this.props;
+            primaryBtnStyle, secondaryBtnStyle, validationErrors, step, totalSteps, children } = this.props;
         const { primaryBtnShowSpinner, secondaryBtnShowSpinner } = this.state;
 
-        return <Modal>
-            <div className="popup">
-                <div className="popup__header">
-                    {headerText}
-                </div>
-
-                <div className="popup__body">
-                    {splitTextByNewLine(bodyText)}
-
-                    <Form
-                        primaryBtnText={primaryBtnText}
-                        primaryBtnStyle={primaryBtnStyle}
-                        onSubmitHandler={this._primaryBtnHandler}
-                        primaryBtnShowSpinner={primaryBtnShowSpinner}
-                        secondaryBtnText={secondaryBtnText}
-                        secondaryBtnStyle={secondaryBtnStyle}
-                        secondaryBtnHandler={this._secondaryBtnHandler}
-                        secondaryBtnShowSpinner={secondaryBtnShowSpinner}
-                        validationErrors={validationErrors}
-                        alwaysEnableSubmitButton={true}
-                    >
-
-                        {children}
-
-                    </Form>
+        return <div className="popup">
```

Comment:
Will this div be positioned in a random place in the page, since I assume the `Modal` is positioned absolute or static?

Reply:
The popup class has no properties, it is only added here to scope the other classes. I like to do this for all components